packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeActionButton.tsx

```diff
@@ -32,10 +40,19 @@ export type FilterActionProps = {
 } & Partial<Pick<ReturnType<typeof useTree>, "getHierarchyLevelDetails">>;
 
 /** @internal */
-export function TreeActionButton({ show, label, action, icon }: TreeItemAction) {
+export function TreeActionButton({ show, label, action, icon, activeDescription }: TreeItemAction) {
   const { localizedStrings } = useLocalizationContext();
   const localizedLabel: string | undefined = localizedStrings[label as keyof typeof localizedStrings];
-  return <Tree.ItemAction visible={show} onClick={action} label={localizedLabel ?? label} icon={icon ?? placeholderIcon} />;
+  const localizedDescription: string | undefined = localizedStrings[activeDescription as keyof typeof localizedStrings];
+  return (
+    <Tree.ItemAction
+      render={<IconButton dot={localizedDescription ?? activeDescription} icon={icon ?? placeholderIcon} variant={"ghost"} label={localizedLabel ?? label} />}
+      label={localizedLabel ?? label}
+      icon={icon ?? placeholderIcon}
```

Comment:
Do we need to set both - `render` and `icon` props? It seems `render` should override whatever you pass to `icon`.

Reply:
This will be fixed by kiwi, because dot is not available for items actions, we need to use render and both IconButton and item action require an icon.